skycultures/chinese/dso_names.zh_CN.fab

```diff
@@ -1,6 +1,5 @@
 NGC 224 |_("奎宿增廿一") 
 NGC 2632 |_("积尸")
-NGC 3372 |_("海山二")
```

Comment:
What is the remaining purpose of the fab files? Names for DSO can be put into the common_names section in index.json.

Reply:
This file (and all the other `.fab` files) can be deleted once there's no information in them that hasn't been incorporated into `index.json`. I just kept them after converting the SC to avoid loss of data.